src/test/scala/org/broadinstitute/dsde/vault/services/uBAMCollection/UBamCollectionDescribeServiceSpec.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.vault.services.uBAMCollection
+
+import org.broadinstitute.dsde.vault.model.uBAMJsonProtocol._
+import org.broadinstitute.dsde.vault.model.{UBamCollection, UBamIngest, UBamIngestResponse,UBamCollectionIngest}
+import org.broadinstitute.dsde.vault.{VaultConfig, VaultFreeSpec}
+import org.broadinstitute.dsde.vault.services.uBAM.UBamIngestService
+import org.broadinstitute.dsde.vault.model.uBAMCollectionJsonProtocol._
+import spray.http.HttpCookie
+import spray.http.HttpHeaders.Cookie
+import spray.http.StatusCodes._
+import spray.httpx.SprayJsonSupport._
+
+class UBamCollectionDescribeServiceSpec extends VaultFreeSpec with UBamCollectionDescribeService with UBamCollectionIngestService with UBamIngestService {
+
+  override val routes = ubamCollectionDescribeRoute ~ uBamIngestRoute
+
+  def actorRefFactory = system
+
+  val openAmResponse = getOpenAmToken.get
+  var testingUbamId = "ubam_invalid_UUID"
+  var testingubamCollectionId = "ubamCollection_invalid_UUID"
+
+
+  val files = Map(("bam", "vault/test/test.bam"), ("bai", "vault/test/test.bai"))
+  val metadata = Map("testAttr" -> "testValue")
+
+  val members = Option(Seq[String](testingUbamId))
+
+  "UBamCollectionDescribeServiceSpec" - {
+
+
+    "while preparing the ubamCollection test data" - {
+      "should successfully store the data using the UBam Ingest path" in {
+        val ubamIngest = new UBamIngest(files, metadata)
+        Post(VaultConfig.Vault.ubamIngestPath, ubamIngest) ~> Cookie(HttpCookie("iPlanetDirectoryPro", openAmResponse.tokenId)) ~> uBamIngestRoute ~> check {
```

Comment:
1. After updating `ubamIngestPath` to be a `def`, pass in the version. To help future developers, the version should originate from a `Table`, like in the [other](https://github.com/broadinstitute/vault-api/blob/f85653b6c20746e50f7aa17af26f110c121182de/src/test/scala/org/broadinstitute/dsde/vault/services/uBAM/UBamIngestServiceSpec.scala#L15) [specs](https://github.com/broadinstitute/vault-api/blob/f85653b6c20746e50f7aa17af26f110c121182de/src/test/scala/org/broadinstitute/dsde/vault/services/uBAM/UBamDescribeServiceSpec.scala#L20), _however_ the version should be an `Int`, not an `Option[Int]`, and should only contain `1`.
2. Use the new `addOpenAmCookie` instead of the old verbose `Cookie` here.

This goes for the other specs as well.